### ISSUE TITLE
[Bug] Application window position problem when opening

### ISSUE THREAD

**Issue by fabiencharrasse** [AppFlowy-IO/AppFlowy#7072]
### Bug Description

Two in one problem:
- The application opens with about 50% off-screen at the top and a little on the right
- The application does not remember the position and the screen in which it was closed.

### How to Reproduce

Nothing special, just install the application

### Expected Behavior

The app should open in the same screen and position where it was last closed.

### Operating System

Linux Mint 22

### AppFlowy Version(s)

0.7.8

### Screenshots

![image](https://github.com/user-attachments/assets/fc3b0b0d-f972-4002-bc09-366656df281a)

I click on the Appflowy icon on the right screen, and it open like in the the screenshot (the top of the screenshot is the top of the left monitor)

### Additional Context

I'm on a 3-screen setup, clicking on the Appflowy icon from the middle screen (the main one) launches the app on the left screen with about 50% off-screen at the top and a little on the middle screen. Putting the app in full screen then closing and reopening it does not solve the problem.

**Comment by LucasXu0** [comment]
@fabiencharrasse Did you try moving the application to the middle screen, closing it, and reopening it?

**Comment by fabiencharrasse** [comment]
> @fabiencharrasse Did you try moving the application to the middle screen, closing it, and reopening it?

> Putting the app in full screen then closing and reopening it does not solve the problem.

Yes. I found the origin of the problem concerning the application that goes out of the screen, I am on a zoom at 175% on the middle screen, if I go to 200%, the application does not go out of the screen and keeps its proportions. 

However, it keeps opening on the left screen instead of the middle screen

**Comment by LucasXu0** [comment]
> However, it keeps opening on the left screen instead of the middle screen

Did you close the application on the middle screen?
